app/views/pieces/new.html.haml

```diff
@@ -1,3 +1,3 @@
 %h1 Add New Music
-%div= render 'form'
+%div.review-container= render 'form'
```

Comment:
`%div.review-container` can be written as `.review-container` since `%div` is implicit